dump/main.py

```diff
@@ -146,19 +145,20 @@ def filter_out_dbs(db_list, collected_info):
     return collected_info
 
 
-def populate_fv(info, module, namespace):
+def populate_fv(info, module, namespace, conn_pool):
     all_dbs = set()
     for id in info.keys():
         for db_name in info[id].keys():
             all_dbs.add(db_name)
 
     db_cfg_file = JsonSource()
-    db_conn = ConnectionPool().initialize_connector(namespace)
     for db_name in all_dbs:
         if db_name == "CONFIG_FILE":
             db_cfg_file.connect(plugins.dump_modules[module].CONFIG_FILE, namespace)
         else:
-            db_conn.connect(db_name)
+            conn_pool.get(db_name, namespace)
+    
+    db_conn = conn_pool.cache.get(namespace, {}).get('conn', None)
```

Comment:
Instead of using 'conn' string, will be good to define a string and use it.
The same for the keys in ConnectionPool like "connected_to" and "conn", as it is used in multiple locations and will be hard to keep track.

Reply:
Updated